eiffel/src/main/java/com/etiennelenhart/eiffel/viewmodel/EiffelViewModel.kt

```diff
@@ -36,12 +46,23 @@ abstract class EiffelViewModel<S : State, A : Action>(
     private val actionDispatcher: CoroutineDispatcher = Dispatchers.Default
 ) : ViewModel() {
 
+    open val debug: Boolean = false
```

Comment:
I agree with the tweak in the logic, however I'm not totally for adding `excludeFromDebug` as a constructor parameter. I feel like it might clutter up the constructor, I'm more partial to overriding values in the body.  And if they wanted they could still override the value in the constructor.

As for it being more discoverable, I think a properly document KDoc will cover that.

That being said if you really want it as a constructor parameter, I'm not totally against it and will make the change if you want.

Reply:
Yeah, good point. I think I'll also move the dispatcher parameters to protected properties later, since they'll probably won't be changed often.